lib/ui/screen-components/home/bank-management/view/main/index.tsx

```
@@ -174,93 +174,112 @@ export default function BankManagementMain() {
+        {/* <ScrollView
+          scrollEnabled={keyboardVisible}
+          contentContainerClassName={`flex flex-col justify-between items-center w-full ${keyboardVisible ? "h-[100%]" : "h-[85%]"} my-6 px-4`}
+        > */}
+        <KeyboardAvoidingView
+          className={`${keyboardVisible ? "h-[100%] " : "h-[85%"}`}
+        >
+          <FlatList
+            data={[
+              <View
+                className={`flex flex-col w-full items-start justify-start  gap-2`}
               >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)